config/default.yml

```
@@ -49,6 +49,21 @@ Rails/ActionFilter:
   Include:
     - app/controllers/**/*.rb
 
+Rails/ActionOrder:
+  Description: 'Enforces consistent ordering of controller actions.'
+  Enabled: false
```

```suggestion
  Enabled: pending
```

New cops are pending by default.